prysm/assetlist.json

```diff
@@ -0,0 +1,41 @@
+{
+  "$schema": "https://github.com/cosmos/chain-registry/blob/master/assetlist.schema.json",
+  "chain_name": "Prysm Network",
+  "assets": [
+    {
+      "description": "Prysm is an Interchain Security Hub",
+      "denom_units": [
+        {
+          "denom": "uprysm",
+          "exponent": 0
+        },
+        {
+          "denom": "prysm",
+          "exponent": 6
+        }
+      ],
+      "base": "uprysm",
+      "name": "Prysm",
+      "display": "Prysm",
+      "symbol": "PRYSM",
+      "logo_URIs": {
+        "png": "https://github.com/LightLABS-Dev/chain-registry/blob/master/prysm/images/prysm%20logo.png",
```

Comment:
this location must point to the Cosmos org's chain registry repo (not the light-labs fork).
Also, prysm logos were already uploaded.